docs/_static/css/custom.css

```diff
@@ -0,0 +1,300 @@
+@font-face {
+  font-family: "Graphik";
+  src: url("https://assets.website-files.com/5dc3b47ddc6c0c2a1af74ad0/5dc3c89751595b2217a8a5ca_Graphik-ExtralightItalic.otf")
```

Comment:
This is the CDN that serves the font files for our corporate website. Because Graphik is a licensed font, we don't want to upload it as part of the Git repo.

Happy to deal with this some other way if you have ideas.

Reply:
Oh, so this is the CDN used by Webflow? I wonder if it will accept non-localhost HTTP requests...

Let's try it out, I guess!

(In the future we should host this elsewhere, but ok for now)